e2e/scripts/testim.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -euox pipefail
+
+main() {
+  local testim_token="$1"
+  if [ -z "$testim_token" ]; then
+    echo "Testim token is required"
+    exit 1
+  fi
+
+  local testim_branch="$2"
+  if [ -z "$testim_branch" ]; then
+    echo "Testim branch is required"
+    exit 1
+  fi
+
+  local test_name="$3"
+  if [ -z "$test_name" ]; then
+    echo "Test name is required"
+    exit 1
+  fi
+
+  echo "Running Testim test: $test_name on branch $testim_branch"
+
+  # port-forward to kotsadm to bypass kurl-proxy
+  kubectl port-forward sts/kotsadm -n kotsadm 3000:3000 &
+  port_forward_pid=$!
```

Comment:
testim CLI uses ngrok to expose locally running services to the test platform. ngrok terminates TLS and this doesn't play nice with kurl-proxy